src/Error/Debugger.php

```diff
@@ -124,10 +124,10 @@ class Debugger
      */
     protected $renderers = [
         // Backwards compatible alias for text that will be deprecated.
-        'txt' => TextRenderer::class,
-        'text' => TextRenderer::class,
+        'txt' => TextErrorRenderer::class,
```

Comment:
I think just having the `txt` key should suffice. We don't have 1:1 name matching for all key => class anyway so the redundant `text` key is not necessary. Everyone understands that `js` stands for "javascript" and similarly `txt` stands for "text".

Reply:
This was my fault. In the first PR, I said why not make this `text` before finding out it's `txt` for BC.